simulation_rate: 1000.0 # Hz
clock_rate: 1000.0 # Hz
realtime_factor: 1.0 # [-]

# when true, the simulation time will get initialized from wall time
sim_time_from_wall_time: true

g: 9.81 # [ms^-2]


iterate_without_input: true

input_timeout: 0.2 # [s]

frames:
  world:
    name: "simulator_origin"
    prefix_with_uav_name: false
  fcu:
    name: "fcu"
    publish_tf: false
  rangefinder:
    name: "garmin"
    publish_tf: true

collisions:
  enabled: true
  crash: true

  # if crash==false, then the collisions are elastic
  rebounce: 100.0

# randomization of the initial conditions around the desired spawn point
randomization:

  enabled: false

  bounds:
    x: 15.0
    y: 15.0
    z: 15.0

ground:
  enabled: true
  z: 0.0 # [m]

# * when enabled, this will place a temporary patch of ground just under the spawn location
# * this patch will disappear after the total_thrust > 0.9*hover_thrust
individual_takeoff_platform:
  enabled: false


start_latitude: 15.2
start_longditude: 14.3


imu_rate: 1000.0 #[Hz]
mag_rate: 50.0 #[Hz]
altitude_rate: 75.0 #[Hz]
position_rate: 10.0 #[Hz]
range_rate: 1000.0 #[Hz]

accel_stddev: 1.0 # [m/s^2] 
accel_bias: 0.0 # [m/s^2]

gyro_stddev: 1.0 # [rad/s]
gyro_bias: 0.0 # [rad/s]

altitude_stddev: 0.0 # [m]
altitude_bias: 0.0 # [m]

mag_stddev: 0.0 # [degrees ?  ]
mag_bias: 0.0 # [degrees ?]

pos_stddev: 0.0 # [m ]
pos_bias: 0.0 # [m]

B_gyro0: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
B_gyro1: [-9.3389408759698e-07, -1.6089487467714587e-06, -1.0604462704379344e-05, -2.1297726134244784e-05, 0.00012950952936099025, 0.0005876133688461059, 0.00012950952936099006, -2.1297726134244777e-05, -1.0604462704379383e-05, -1.6089487467714314e-06, -9.338940875969855e-07 ]
B_gyro2: [-1.3773530287800962e-06, -1.992805573328216e-06, -1.1517570411651698e-05, -2.391038136960739e-05, 0.00015662811384959942, 0.0006492581042933454, 0.0001566281138495992, -2.3910381369607365e-05, -1.1517570411651738e-05, -1.9928055733281863e-06, -1.3773530287801042e-06 ]
B_accel0: [-1.5387658426499426e-05, -3.105787748924258e-06, -8.777057144170397e-05, -0.00021566799013816843, 0.002625289446753541, 0.007262249910002694, 0.002625289446753539, -0.00021566799013816875, -8.777057144170412e-05, -3.1057877489240393e-06, -1.538765842649947e-05 ]
B_accel1: [-1.470183810365154e-05, -1.8600687630721746e-05, -0.00012024162913017457, -0.00032691304069366227, 0.002382428869523561, 0.00698719283286576, 0.0023824288695235593, -0.0003269130406936627, -0.00012024162913017474, -1.8600687630721513e-05, -1.470183810365159e-05 ]
B_accel2: [2.2126263206915357e-05, 4.6718497383721926e-05, 0.00017308853634680148, -5.3232293546626156e-05, 0.003023526726490537, 0.008597122529460415, 0.0030235267264905334, -5.323229354662634e-05, 0.00017308853634680115, 4.671849738372211e-05, 2.21262632069153e-05 ]

B_position0: [1]
B_position1: [1]
B_position2: [1]

B_mag0: [1]
B_mag1: [1]
B_mag2: [1]

B_range: [1]

B_altitude: [1]